src/app/core-ui/main/announce-notification/announcement-notification.component.html

```diff
@@ -0,0 +1,25 @@
+<!-- Replace with the new changes that allien made on different branch -->
+<!-- <mat-card class="notification info" *ngIf="latestClientVersion && isNewUpdateAvailable()">
+  <mat-card-title class="title">
+    New update available <span class="tag">v{{ latestClientVersion }}</span> &ndash; please upgrade:
+    <a href="{{releaseUrl}}" target="_blank" class="button" mat-button color="primary">
+      <mat-icon class="icon" fontSet="partIcon" fontIcon="part-lightning"></mat-icon>
+      Upgrade now
+    </a>
+  </mat-card-title>
+</mat-card> --><!-- .notification -->
+
+
+<!-- Alpha mainnet warning -->
+<mat-card class="notification alert" *ngIf="isAlpha()"><!-- *ngIf="" -->
+  <mat-card-title class="title">
+    The Particl Marketplace alpha is still in development and not 100% private yet &ndash; use it at your own risk!
+    <p *ngIf="latestClientVersion === currentClientVersion">You are on the latest version</p>
+
+    <p *ngIf="isNewUpdateAvailable()">You are on an outdated version, please click to update</p>
```

Comment:
Is any of this HTML necessary, if the component exists to inform users of new messages (or currently just the alpha/beta warning which has already been included above)? This can likely be removed.